documentation/wiki/Building-Testing-and-Debugging-on-Full-Framework-MSBuild.md

```diff
@@ -63,7 +63,7 @@ Please see [Contributing Code](https://github.com/dotnet/msbuild/blob/main/docum
 To build projects using the MSBuild binaries from the repository, you first need to do a build which produces
 a "bootstrap" directory. The "bootstrap" directory mimics a Visual Studio installation by acquiring additional
 dependencies (Roslyn compilers, NuGet, etc.) from packages or from your local machine (e.g. props/targets
-from Visual Studio). This will happen by default when running `.\build.cmd`. The bootstrap can be disabled by running `.\build.cmd /p:CreateBootstrap=false`.
```

Comment:
Do we want to remove this?
The statement seems to be true: 

https://github.com/dotnet/msbuild/blob/c4d51a11b84b4aabd9e5da1e3099f2c7c85024fe/eng/BootStrapMsBuild.targets#L20-L22